doc/reference/reference_lua/box_index/tuple_pos.rst

```diff
@@ -0,0 +1,51 @@
+.. _box_index-tuple_pos:
```

Comment:
`tuple_pos` does not work with functional and multi key indexes. I think we could place it on this page as a `Note` section.

Reply:
thanks, fixed